receiver/dcgmreceiver/scraper_gpu_test.go

```diff
@@ -61,6 +61,9 @@ func TestScrapeWithGpuPresent(t *testing.T) {
 	err := scraper.start(context.Background(), componenttest.NewNopHost())
 	require.NoError(t, err)
 
+	// Make sure we wait long enough to do multiple collections.
+	time.Sleep(10 * time.Second)
```

Comment:
Should this be a constant somewhere?

Reply:
It's only used in the test, so it's fine for it to be a constant. But, there's a broader point that the test should be more explicit about what it's checking. I refactored this into two separate tests, one of which explicitly checks to make sure we're not collecting too often.